Shoko.Server/Models/SVR_AnimeEpisode.cs

```diff
@@ -146,7 +146,39 @@ public List<CL_VideoDetailed> GetVideoDetailedContracts(int userID)
 
         public CL_AnimeEpisode_User GetUserContract(int userID, ISessionWrapper session = null)
         {
-            return GetOrCreateUserRecord(userID, session).Contract;
+            var anidbEpisode = AniDB_Episode;
+            var seriesUserRecord = RepoFactory.AnimeSeries_User.GetByUserAndSeriesID(userID, AnimeSeriesID);
+            var episodeUserRecord = GetOrCreateUserRecord(userID, session);
+            var englishTitle = RepoFactory.AniDB_Episode_Title.GetByEpisodeIDAndLanguage(AniDB_EpisodeID, "EN")
+                .FirstOrDefault()?.Title;
+            var romajiTitle = RepoFactory.AniDB_Episode_Title.GetByEpisodeIDAndLanguage(AniDB_EpisodeID, "X-JAT")
+                .FirstOrDefault()?.Title;
```

Comment:
Best to inline where you can, and it looks like the last two aren't used.

Reply:
Forgot to remove them :sweat_smile: